### PR TITLE
Realign hover effect

### PR DESCRIPTION
**Pull request by timuric**
Hover effect with better alignment. Requires test fix.

**PR intended to be tested with API branch:** 

### Screenshots
image

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

File: src/products/components/ProductTile/ProductTile.tsx [hunk 1]
```diff
@@ -15,66 +16,92 @@ const commonThumbnailProps = {
...
+          color="textNeutralDefault"
+          variant="bodyEmp"
+          size="small"
+        >
+          {product.name}
+        </Text>
       </Box>
+      <Box
+        position={"absolute"}
+        style={{
```
Comment (krzysztofzuraw) [inline src/products/components/ProductTile/ProductTile.tsx:92]:
```suggestion
        margin="auto"
        __right={`calc(${vars.space[2]} / -2)`}
        __top={`calc(${vars.space[2]} / -2)`}
        __width={`calc(100% + ${vars.space[2]})`}
        __height={`calc(100% + ${vars.space[2]})`}
        __opactiy={0.1}
        
```

File: src/products/components/ProductTile/ProductTile.tsx [hunk 2]
```diff
@@ -15,66 +16,92 @@ const commonThumbnailProps = {
...
-        backgroundColor="surfaceNeutralSubdued"
-      >
-        <Box __width="40%" __height="40%">
-          <ProductsIcons
-            size="fill"
-            color="iconNeutralDisabled"
-            data-test-id={`placeholder-svg-${product.id}`}
-          />
+}) => {
+  const { spacing } = useTheme();
```
Comment (krzysztofzuraw) [inline src/products/components/ProductTile/ProductTile.tsx:28]: **suggestion**: if you migrate to `vars` from `@salero/macaw-ui/next` (`import { vars } from "@saleor/macaw-ui/next";` you shouldn't need `spacing` here

**Review by krzysztofzuraw — APPROVED** [review]

File: src/components/AppLayout/ListFilters/ListFilters.tsx [hunk 3]
```diff
@@ -33,9 +33,6 @@ export const ListFilters = ({
       gap={7}
       paddingBottom={5}
       paddingX={9}
-      borderColor="neutralPlain"
-      borderBottomStyle="solid"
-      borderBottomWidth={1}
```
Comment (poulch) [inline src/components/AppLayout/ListFilters/ListFilters.tsx:38]: question: don't we need this border anymore?
Screenshot 2023-04-21 at 11 30 40

File: src/products/components/ProductListTiles/ProductListTiles.tsx [hunk 4]
```diff
@@ -42,6 +42,7 @@ export const ProductListTiles: React.FC<ProductListTilesProps> = ({
           gridTemplateColumns={{ mobile: 3, tablet: 5, desktop: 6 }}
           gap={9}
           padding={9}
+          __paddingTop={`calc(${vars.space[9]} - ${vars.space[5]}`}
```
Comment (poulch) [inline src/products/components/ProductListTiles/ProductListTiles.tsx:45]: question: why not use space 4?
```suggestion
         paddingTop={4}

File: src/components/AppLayout/ListFilters/ListFilters.tsx (hunk 3 above)
Comment (timuric) [reply src/components/AppLayout/ListFilters/ListFilters.tsx:38]: @poulch this border is redundant

File: src/products/components/ProductListTiles/ProductListTiles.tsx (hunk 4 above)
Comment (timuric) [reply src/products/components/ProductListTiles/ProductListTiles.tsx:45]: @poulch space 4 is not equal to space `9-5`, as they follow a non-linear scale, it is a necessary offset calculation to make surrounding equal

**Review by poulch — APPROVED** [review]

**Comment by wojteknowacki** [comment]
Checked by QA

**Pull request merged**